docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-3.md

```diff
@@ -15,26 +15,24 @@ use the following commands to launch the placement translator; it
 requires the ESPW to be current at start time.
 
 ```shell
-(
-  KUBECONFIG=$SM_CONFIG placement-translator -v=4  &> /tmp/placement-translator.log &
-  sleep 10
-)
+KUBECONFIG=$SM_CONFIG placement-translator -v=4  &> /tmp/placement-translator.log &
+sleep 10
 ```
 
 The following commands wait for the placement translator to get its
 job done for this example.
 
 ```shell
-FLORIN_MB_CONFIG="${PWD}/temp-space-config/${FLORIN_SPACE}"
-GUILDER_MB_CONFIG="${PWD}/temp-space-config/${GUILDER_SPACE}"
-kubectl-kubestellar-get-config-for-space --space-name $FLORIN_SPACE --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --output $FLORIN_MB_CONFIG
-kubectl-kubestellar-get-config-for-space --space-name $GUILDER_SPACE --sm-core-config $SM_CONFIG --sm-context $SM_CONTEXT --output $GUILDER_MB_CONFIG
+FLORIN_MB_KUBECONFIG="${PWD}/${FLORIN_SPACE}.kubeconfig"
+GUILDER_MB_KUBECONFIG="${PWD}/${GUILDER_SPACE}.kubeconfig"
+kubectl-kubestellar-space-get_kubeconfig $FLORIN_SPACE --kubeconfig $SM_CONFIG $in_cluster $FLORIN_MB_KUBECONFIG
+kubectl-kubestellar-space-get_kubeconfig $GUILDER_SPACE --kubeconfig $SM_CONFIG $in_cluster $GUILDER_MB_KUBECONFIG
 
 # wait until SyncerConfig, ReplicaSets and Deployments are ready
 mbxws=($FLORIN_SPACE $GUILDER_SPACE)
 for ii in "${mbxws[@]}"; do
   (
-    export KUBECONFIG="${PWD}/temp-space-config/$ii"
+    export KUBECONFIG="${PWD}/${ii}.kubeconfig"
```

Comment:
Do we need to export in this case?  Each kubectl statement in the body of the for loop would still get the correct KUBECONFIG even if it was local and there is no call to a sub script.

Reply:
You seem to be assuming that `KUBECONFIG` is automatically exported even if export is not explicitly requested here. You may have forgotten that prior to this PR there was another command that marked `KUBECONFIG` for export. It was very early in example1 and easy to forget about later. In this PR I am removing that, as it is a confusing long-range dependency. So in this subshell (the round brackets you see here) it is necessary to either `export KUBECONFIG=...` as a standalone command or specify `KUBECONFIG=...` on each `kubectl ...` command.